src/item.cpp

```diff
@@ -1635,6 +1630,24 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
                                e.obj().name().c_str(), e.obj().description().c_str() ) );
         }
 
+        // does the item fit in any holsters?
+        auto holsters = Item_factory::find( [this]( const itype &e ) {
+            if( !e.can_use( "holster" ) ) {
+                return false;
+            }
+            auto ptr = dynamic_cast<const holster_actor *>( e.get_use( "holster" )->get_actor_ptr() );
+            return ptr->can_holster( *this );
+        } );
+
+        if( !holsters.empty() ) {
+            std::vector<std::string> opts;
+            std::transform( holsters.begin(), holsters.end(), std::back_inserter( opts ), []( const itype *e ) {
+                return e->nname( 1 );
+            } );
+            insert_separation_line();
+            info.emplace_back( "DESCRIPTION", _( "<bold>Can be stored in:</bold> " ) + enumerate_as_string( opts ) );
```

Comment:
You don't need a temporary vector for this. There's an overloaded version of `enumerate_as_string()`:
```c++
enumerate_as_string( holsters.begin(), holsters.end(), []( const itype *e ) {
    return e->nname( 1 );
} );
```